openacademy/models/openacademy_course.py

```diff
@@ -10,8 +11,13 @@ class Course(models.Model):
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
 
     _sql_constraints = [
-        ("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."),
-        ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")
+        ("description_and_title_different",
```

Comment:
Same here, this is an unrelated change

Reply:
In the same PR where the REF was requested, these changes were also requested. I believe.

Reply:
Yes, I asked both things, but I meant separated PRs for each of them.

Having changes separated is important to make commits granularized. In case one commit needs to be reverted, only changes that actually correspond to the reverted feature will be affected.

Reply:
Okay, got it. I already made a new PR with requesting the REF:

https://github.com/ejdecena/openacademy/pull/12